### ISSUE TITLE
Toast not working when running from app directory.

### ISSUE THREAD

**Issue by nles** [tamagui/tamagui#1380]
### Current Behavior

When using the `next-expo-solito` template with `app` directory, toasts will not work on web. `useToastController` seems to return an empty object. Same thing works just fine when running from `pages` directory.

I also noticed that `@tamagui/config` is needed for the `app` directory version, but not included in the package.

### Tamagui Version

```markdown
latest (1.38.1)
```

### Reproduction

```markdown
See reproduction in this CodeSandbox:

https://codesandbox.io/p/sandbox/sad-panini-z7zjsm?file=%2Fapps%2Fnext%2Fnext.config.js%3A12%2C1

What I did:
- changed `experimental.appDir` to `true` in `next.config.js`
- got rid of `pages` directory (renamed to `_pages`)
- installed `@tamagui/config` into `apps/next` (that package was missing, so `TamaguiProvider.ts` would not work)
- commented out `Solito` related things that would also break within the `app` directory since the latest stable version (3) does not support it
```

**Comment by deggertsen** [comment]
Second. I've run into the same issue where useToastController returns an empty object making it so .show() is undefined.

**Comment by ppold** [comment]
bump, I'm also encountering this

**Comment by StefanEissler** [comment]
For me also the @tamagui/conifg is not importing correctly

**Comment by gorbypark** [comment]
Bump / still an issue in 1.75.7

**Comment by gorbypark** [comment]
I believe I found the issue.  In the file `apps/next/app/TamaguiProvider.tsx` the provider is different from the provider used in the pages version of things.  After working backwards through the labyrinth of nested abstractions, I didn't see the toast provider anywhere in there, which would explain the .show() is undefined error.

In the end, I just imported and used the same provider component the pages dir version was using.  

So, in `apps/next/app/TamaguiProvider.tsx` just `import { Provider } from 'app/provider'` and then comment out/remove the TamaguiProviderOG usage and pass in the newly imported provider.  Here's an example of the return of `apps/next/app/TamaguiProvider.tsx`
```tsx
  return (
    <NextThemeProvider
      onChangeTheme={(next) => {
        setTheme(next as any)
      }}
    >
      <Provider disableRootThemeClass defaultTheme={theme}>
        {/* <TamaguiProviderOG config={config} themeClassNameOnRoot defaultTheme={theme}> */}
        {children}
        {/* </TamaguiProviderOG> */}
      </Provider>
    </NextThemeProvider>
  )
}
```
